src/skins/vector/css/matrix-react-sdk/views/elements/_RichText.scss

```diff
@@ -24,9 +25,10 @@
     padding-right: 5px;
 }
 
-.mx_EventTile_highlight .mx_EventTile_content .markdown-body a.mx_UserPill_me {
-    color: $accent-fg-color;
-    background-color: $mention-user-pill-bg-color;
+.mx_EventTile_highlight .mx_EventTile_content .markdown-body a.mx_UserPill_me, .mx_AtRoomPill {
```

Comment:
I think if you're more specific with the CSS selector, you can get away without the !important:
```css
.mx_EventTile_highlight .mx_EventTile_content .markdown-body a.mx_UserPill_me, 
.mx_EventTile_highlight .mx_EventTile_content .markdown-body a.mx_AtRoomPill {
```

Reply:
ah yes! good point